assets/javascripts/discourse/components/ai-persona-editor.gjs

```diff
@@ -433,20 +379,56 @@ export default class PersonaEditor extends Component {
           disabled={{this.editingModel.system}}
         />
       </div>
-      <div class="control-group">
-        <label>{{I18n.t "discourse_ai.ai_persona.max_context_posts"}}</label>
-        <Input
-          @type="number"
-          lang="en"
-          class="ai-persona-editor__max_context_posts"
-          @value={{this.editingModel.max_context_posts}}
+      {{#if this.editingModel.user}}
+        {{#if this.chatPluginEnabled}}
+          <div class="control-group ai-persona-editor__allow_chat">
+            <label>
+              <Input
+                @type="checkbox"
+                @checked={{this.editingModel.allow_chat}}
+              />
+              {{I18n.t "discourse_ai.ai_persona.allow_chat"}}</label>
+            <DTooltip
+              @icon="question-circle"
+              @content={{I18n.t "discourse_ai.ai_persona.allow_chat_help"}}
+            />
+          </div>
+        {{/if}}
+        <div class="control-group ai-persona-editor__mentionable">
+          <label>
+            <Input
+              @type="checkbox"
+              @checked={{this.editingModel.mentionable}}
+            />
+            {{I18n.t "discourse_ai.ai_persona.mentionable"}}</label>
+          <DTooltip
+            @icon="question-circle"
+            @content={{I18n.t "discourse_ai.ai_persona.mentionable_help"}}
+          />
+        </div>
+      {{/if}}
+      <div class="control-group ai-persona-editor__tool-details">
+        <label>
+          <Input @type="checkbox" @checked={{this.editingModel.tool_details}} />
+          {{I18n.t "discourse_ai.ai_persona.tool_details"}}</label>
+        <DTooltip
+          @icon="question-circle"
+          @content={{I18n.t "discourse_ai.ai_persona.tool_details"}}
```

Comment:
```suggestion
          @content={{I18n.t "discourse_ai.ai_persona.tool_details_help"}}
```

Minor oversight, the tip is the same as the label atm but you have the help tip defined in en.yml